code/backend/batch/utilities/helpers/AzureBlobStorageHelper.py

```diff
@@ -101,16 +103,26 @@ def file_exists(self, file_name):
 
         return blob_client.exists()
 
-    def upload_file(self, bytes_data, file_name, content_type="application/pdf"):
+    def upload_file(self, bytes_data, file_name, content_type: Optional[str] = None):
         # Create a blob client using the local file name as the name for the blob
         blob_client = self.blob_service_client.get_blob_client(
             container=self.container_name, blob=file_name
         )
+
+        if content_type is None:
```

Comment:
Can you add a test where this is **not** none. I think it would currently leave the `charset` as `None` and I'm not sure what python does when you try to add `None` to a string.

Reply:
Good catch! Fixed and added a test